domains/sharath.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Social media website project",
+  "repo": "https://github.com/thesct22",
+  "owner": {
+    "username": "thesct22",
+    "email": "sharathct22@gmail.com"
+  },
+  "record": {
+    "CNAME":"http://34.67.228.107.xip.io"
```

This is not a valid cname.

Maybe this is what you wanted to do?

```suggestion
    "CNAME":"34.67.228.107.xip.io"
```